src/utilities/speech.ts

```
@@ -28,18 +31,31 @@ function set_voices(): void {
 	}
 }
 
-export function speak(text: string): void {
+/**
+ * Read out loud the given text.
+ * @param text - The text to read out loud.
+ */
+export function read_out_loud(text: string, options: { onend?: () => void } = {}): void {
 	if (!supports_speech()) return undefined;
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)